src/helpers/strings-de.ts

```diff
@@ -56,6 +56,7 @@ let strings: { [key: string]: string } = {
     "support": "Support",
     "all-notifications": "Alle Benachrichtigungen",
     "steps": "Schritte",
+    "steps-with-google-fit": "Schritte (inkl. Google Fit)",
```

Comment:
This is a participant-facing label actually, so it should still just be "Steps" actually...since they do not care

Reply:
Ah.  I forgot about that.  We also use this label in the VB data type select dropdown, which is why I was making it different.

I'll update this to use the existing "Steps" label and then add an override of some kind in VB to handle that situation.